rocketry/test/conftest.py

```diff
@@ -80,7 +80,7 @@ def session():
         "task_execution": "process",
     }, delete_existing_loggers=True)
     rocketry.session = session
-    session.set_as_default()
+    # session.set_as_default()
```

Comment:
I think the tests fail because of this. I'll investigate this further but I think what you did here should be done as the default sessions should be eventually removed.

Reply:
Inspecting the tests that failed, I think the fault is just in the tests. This failure ``rocketry/test/condition/test_core.py:62: AssertionError`` relies on the boolean operation of the conditions which is not actually used by the internals but this uses the defaults as you cannot pass arguments to ``__bool__`` except the object itself.